### DISCUSSION TITLE
Allow disabling of manual deployment invocations.

### DISCUSSION THREAD

**Discussion by aaazzam** [PrefectHQ/prefect#14853] [Ideas]
x-posted from community. 

Denying manual invocations of a deployment takes _work_, like inspecting the invocation a la `get_client` and failing it if it doesn't contain an `auto-scheduled` tag. Pausing a deployment shuts off _all_ invocation methods, so there isn't a happy path for someone who wants to run code _only_ on a schedule. 

Idea: Allow a user to disable all manual invocations of a deployment while still allowing temporal/automation-based triggers.

**Comment by cicdw** [comment]
Interesting - to me that _sounds_ like a pattern that is best enforced through permissions, not business logic. What's the business logic/use case motivation for such a constraint?
